src/main/java/org/asciidoc/intellij/editor/javafx/JavaFxHtmlPanel.java

```diff
@@ -61,6 +61,11 @@ protected String compute() {
         .append("<script src=\"").append(clazz.getResource("scrollToElement.js")).append("\"></script>\n")
         .append("<script src=\"").append(clazz.getResource("processLinks.js")).append("\"></script>\n")
         .append("<script src=\"").append(clazz.getResource("pickSourceLine.js")).append("\"></script>\n")
+        .append("<script src=\"").append(clazz.getResource("mathJaxConfig.js")).append("\"></script>\n")
```

Comment:
I didn't find mathJaxConfig.js here, only localMathJaxConfig.js. Could mathJaxConfig.js be removed, of the other file renamed?

Reply:
Ups, L64 is a remain of my attempts to load the MathJax configuration from a file.
I thought I got it using the config URL option but I was wrong. After remove L64 (a line with no sense), MathJax doesn't work.

So, I have made a commit where the content of localMathJaxConfig.js is in-line.